code/modules/modular_computers/file_system/programs/command/comm.dm

```diff
@@ -207,7 +208,7 @@
 					var/old_level = security_level
 					if(!current_level) current_level = SEC_LEVEL_GREEN
 					if(current_level < SEC_LEVEL_GREEN) current_level = SEC_LEVEL_GREEN
-					if(current_level > SEC_LEVEL_BLUE) current_level = SEC_LEVEL_BLUE //Cannot engage delta with this
+					if(current_level > SEC_LEVEL_YELLOW) current_level = SEC_LEVEL_BLUE //Cannot engage delta with this
```

Comment:
Should this be `current_level = SEC_LEVEL_BLUE`, or `current_level = SEC_LEVEL_YELLOW`? I'm not too sure of the logic here, it might be fine.

Reply:
This is for people who try and do funny things with NanoUI. It prevents them from doing those things.